Source/WebCore/css/parser/SizesCalcParser.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright 2014-2017 The Chromium Authors. All rights reserved.
-// Copyright (C) 2016 Apple Inc. All rights reserved.
+// Copyright (C) 2024 Apple Inc. All rights reserved.
```

Comment:
Why are you removing the older date? That’s not correct. Most likely should write 2016-2024.